.buildkite/pipeline.provision.yml

```diff
@@ -14,6 +14,10 @@ steps:
           command: preview
           project_dir: pulumi/grapl
           stack: grapl/testing
+      - grapl-security/pulumi#v0.1.2:
```

Comment:
This will work, because the plugin runs `pulumi` in its [command hook](https://github.com/grapl-security/pulumi-buildkite-plugin/blob/main/hooks/command), and a Buildkite job can only have [one command](https://buildkite.com/docs/agent/v3/hooks#job-lifecycle-hooks).

You'll need to specify this second run in a job of its own. If it depends on `grapl/testing` having run first, then that job will need to use [depends_on](https://buildkite.com/docs/pipelines/dependencies#defining-explicit-dependencies) or [wait](https://buildkite.com/docs/pipelines/wait-step) to reflect that.

Reply:
i assume this is supposed to be a "this wont work"

Reply:
🤦 Ugh, correct... this will _not_ work.